tests/test_report_builder.py

```diff
@@ -70,6 +71,16 @@ def test_invalid_json(self):
         with self.assertRaises(Exception):
             ReportBuilder([invalid_json_path], verbose=1)
 
+    def test_corrupted_json(self):
+        with tempfile.TemporaryDirectory() as tmpdir:
+            corrupted_json_path = os.path.join(os.path.dirname(__file__), "data", "fib.py")
+            valid_json_path = os.path.join(os.path.dirname(__file__), "data", "multithread.json")
+            corrupted_json_file = shutil.copy(corrupted_json_path, os.path.join(tmpdir, "corrupted.json"))
+            valid_json_file = shutil.copy(valid_json_path, os.path.join(tmpdir, "valid.json"))
+            rb = ReportBuilder([corrupted_json_file, valid_json_file], verbose=1)
+            with io.StringIO() as s:
+                rb.save(s)
```

Comment:
We should check for some printed messages here. This test only assures no exceptions raised.